distributed/cli/tests/test_dask_worker.py

```diff
@@ -280,6 +282,17 @@ def test_nprocs_expands_name(loop):
                     assert len(set(names)) == 4
 
 
+def test_nprocs_deprecation():
+
+    runner = CliRunner()
+    with pytest.warns(UserWarning, match="nprocs"):
```

Comment:
This checks that the warning message emitted is slightly more meaningful 

```suggestion
    with pytest.warns(FutureWarning, match="renamed to --num-workers"):
```